source/js/ng-img-crop.js

```diff
@@ -160,17 +160,37 @@ crop.directive('imgCrop', ['$timeout', 'cropHost', 'cropPubSub', function ($time
                 };
             };
 
-            // Will get the users language settings, and return the appropriate loading text.
+            // Will get the users language settings, and return the appropriate loading text
             var printLoadMsg = function() {
                 var language = window.navigator.userLanguage || window.navigator.language;
-                var msg = 'Loading'; // Default to English
-                if(language.contains('fr')) { // French Msg.
-                    msg = 'Chargement';
+                
+                switch(language) {
+                    case 'nl':
+                        return 'Aan het laden';
+                    break;
+                        
+                    case 'fr':
```

Comment:
Correct me if i'm wrong, but im not sure this function will always return full strings. For example, various returns for English can be both `en` and `en-US`?

[Mozilla Docs on NavigatorLanguage.language](https://developer.mozilla.org/en-US/docs/Web/API/NavigatorLanguage/language)